packages/components/src/styles/table-cell/table-cell.css

```diff
@@ -5,6 +5,56 @@
     @apply border-r-[1px];
   }
 
+  &--shadow-bottom {
+    &:after {
+      @apply opacity-0 absolute pointer-events-none transition-opacity duration-1000 ease-out	left-0 right-0 h-2.5 bottom-[-10px] content-[''] bg-gradient-to-b from-neutral-300 to-transparent;
+    }
+
+    &.sd-table-cell--shadow-active:after {
+      @apply opacity-[100%];
+    }
+
+    /* fix that shows line on top of table-cell */
+    &:before {
+      @apply bg-neutral-400 block content-[''] absolute top-[-1px] w-full h-[1px] left-0;
+    }
+  }
+
+  &--shadow-right {
+    &:after {
+      @apply opacity-0 absolute pointer-events-none transition-opacity duration-1000 content-[''] bg-gradient-to-r from-neutral-300 to-transparent top-0 bottom-0 right-[-10px] w-2.5;
```

Comment:
just out of curiosity: cant we combine those classes into a single class we can use for other shadows in other components? 

I see `opacity-0 absolute pointer-events-none transition-opacity duration-1000 from-neutral-300 to-transparent` as duplicates in here, which only describe the kind of shadow we want to have (not sure about the `absolute`).

Feels like we will have the same with other shadows or is this the only use case for a shadow?

Reply:
@coraliefeil do you see an upcoming reuse of this type of transition?

Reply:
@mariohamann should we create a token for this, or how to deal with that?

Reply:
refactored with the suggestions from @mariohamann